recipes/natural_language_processing/rag/Makefile

```diff
@@ -0,0 +1,39 @@
+APP=rag
+MODELIMAGE ?= quay.io/ai-lab/mistral-7b-instruct:latest
+APPIMAGE ?= quay.io/ai-lab/${APP}:latest
+SERVERIMAGE ?= quay.io/ai-lab/llamacpp-python:latest
+SSHPUBKEY ?= $(shell cat ${HOME}/.ssh/id_rsa.pub;)
+.PHONY: build
+build:
+	podman build -f builds/Containerfile -t ghcr.io/ai-lab-recipes/${APP} .
+
+.PHONY: bootc
+bootc:
+	sudo podman build --cap-add SYS_ADMIN --build-arg "SSHPUBKEY=$(SSHPUBKEY)" -f bootc/Containerfile -t ghcr.io/ai-lab-recipes/${APP}-bootc .
+
+.PHONY: quadlet
+quadlet:
+# Modify quadlet files to match the server, model and app image
+	sed -e "s|SERVERIMAGE|${SERVERIMAGE}|" \
+	    -e "s|APPIMAGE|${APPIMAGE}|g" \
+	    -e "s|MODELIMAGE|${MODELIMAGE}|g" \
+	    quadlet/${APP}.image \
+	    > /tmp/${APP}.image
```

Comment:
Hmm, what's up with the use of `/tmp` here?  Predictable filenames in `/tmp` are not good (https://cwe.mitre.org/data/definitions/377.html).

Reply:
Good point, I will clean this up.